client/components/data/checkout/index.jsx

```diff
@@ -1,33 +0,0 @@
-/**
- * External dependencies
- */
-
-import React from 'react';
-
-/**
- * Internal dependencies
- */
-import StoreConnection from 'calypso/components/data/store-connection';
-import CartStore from 'calypso/lib/cart/store';
-import TransactionStore from 'calypso/lib/transaction/store';
```

Comment:
It looks like this is the only time this import was used. Would anything else use this?

Reply:
Wow, you're right. And not only that, but it looks like the entirety of that directory might be unused. I will verify and then trash it...